packages/network/src/protocol/TrackerServer.ts

```diff
@@ -69,24 +69,28 @@ export class TrackerServer extends EventEmitter {
     sendRtcOffer(
         receiverNodeId: string, 
         requestId: string, 
-        originatorInfo: TrackerLayer.Originator, 
+        originatorInfo: TrackerLayer.Originator,
+        connectionId: string, 
         description: string
     ): Promise<TrackerLayer.RelayMessage> {
-        return this.send(receiverNodeId, new TrackerLayer.RelayMessage({
+        const msg = new TrackerLayer.RelayMessage({
```

Comment:
Very stylistic nitpick: I'd keep this in the same style as before, i.e.
```ts
return this.send(receiverNodeId, new TrackerLayer.RelayMessage({
...
}
```
mostly because it would match the rest of the methods in this class.

Reply:
ok